pdns/dnsdistdist/docs/changelog.rst

```diff
@@ -1,6 +1,95 @@
 Changelog
 =========
 
+.. changelog::
+  :version: 1.9.7
+  :released: 4th of October 2024
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 14638
+    :tickets: 14562
+
+    Fix build with boost 1.86.0
+
+  .. change::
+    :tags: Bug Fixes, DNS over TLS
+    :pullreq: 14639
+    :tickets: 14631
+
+    Fix handling of proxy protocol payload outside of TLS for DoT
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 14641
+    :tickets: 14568
+
+    Stop reporting timeouts in ``topSlow()``, add ``topTimeouts()``
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 14643
+    :tickets: 14552
+
+    Return a valid unix timestamp for Dynamic Block's ``until``
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 14644
+    :tickets: 14558
+
+    Fix EDNS flags confusion when editing the OPT header
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 14645
+    :tickets: 14549
+
+    Fix compilation with GCC 15 (Holger Hoffstätte)
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 14646
+
+    Add warnings about large values passed to ``setMaxTCPClientThreads``
+
+  .. change::
+    :tags: Improvements, DNS over QUIC, DNS over HTTP3
+    :pullreq: 14647
+
+    Update Quiche to 0.22.0 (in our packages)
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 14640
+
+    Handle a non-existent default pool when removing a server
```

Comment:
I'm afraid I don't understand why this pattern is denied. I'm not a native speaker but I see `non-existent` in the Cambridge and Collins dictionary so I guess I'm missing something?

Reply:
The spell-checker itself treats `-` as a word boundary and both `non` and `existent` are in the dictionary, so it can't say the "word" is misspelled. The [rule in question is](https://github.com/PowerDNS/pdns/blob/3f0cb1a2e3a7ae0d367d87f7cb4eb5633505bb76/.github/actions/spell-check/line_forbidden.patterns#L73-L75), indeed, designed to encourage en-US spelling. (The spell-checker without the rule isn't _that_ biased to US, the rules are supplemental and could be removed.)

Reply:
OK, thanks.